api/question/schema_query.py

```diff
@@ -25,6 +25,12 @@ class Query(graphene.ObjectType):
                                        page=graphene.Int(),
                                        per_page=graphene.Int())
 
+    filter_rooms_by_responses = graphene.Field(
+        Responses,
+        lower_bound=graphene.Int(),
+        upper_bound=graphene.Int(),
```

Comment:
Can you please use a more descriptive variable name? I have been trying hard to understand what you mean by upper bound and lower bound